src/lib/searchInputFilters.js

```diff
@@ -0,0 +1,9 @@
+import { MODEL_SEARCH_FIELDS } from './consts';
+
+export function searchInputFilters(modelName, filter) {
+  if (!filter) return {};
+  const filters = [];
+  const fields = MODEL_SEARCH_FIELDS[modelName];
+  fields.forEach(field => filters.push({ [field]: { contains: filter, mode: 'insensitive' } }));
```

Comment:
🟠 please prefer using `map` instead of `forEach + mutation`

Reply:
I`ll change it